### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by eli-lim**
Enhances github profile readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ### Hi there 👋
 
+My name is Chloe! I'm a Software Developer! 🌝🤏
+
+
```
Comment (eli-lim) [inline README.md:8]:
```suggestion
<sup><b>Languages</b></sup> <br>
<img alt="Typescript" src="https://img.shields.io/badge/typescript-%23007ACC.svg?style=for-the-badge&logo=typescript&logoColor=white" height="25" />
<img alt="Python" src="https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54" height="25" />

```

**Comment by weiyuan95** [comment]
LGTM! :shipit:

**Comment by chloezxyy** [comment]
Thanks for the eyes!

**Comment by chanyusiang** [comment]
> Thanks for the eyes!

123 copy

**Pull request merged**